lab/app.js

```diff
@@ -0,0 +1,43 @@
+const express = require("express")
+const cors = require("cors")
+const port = 3000
+const app = express()
+app.use(cors())
+
+const animalObject = 
+{
+    animals: ["zebra","dog","lion"],
+    verdict: "",
+
+
+}
+
+const isAnimal =(req,res,next)=>{
+        if(animalObject.animals.includes(req.params.animalid)){ 
+            animalObject.verdict = "true"
```

Comment:
This should be a boolean, not a string. 

Simply remove the string quotations to turn the string into a boolean

Booleans are just `true` and `false`